utils/deps_xml.py

```diff
@@ -16,29 +16,37 @@
 parser = argparse.ArgumentParser()
 parser.add_argument(
     "inputfile", type=argparse.FileType('r'), help="Input XML file")
+parser.add_argument(
+    "--file_per_line", action='store_true', help="Output dependencies file per line, rather than Make .d format.")
```

Comment:
Maybe we just drop the `Make .d` format (or log an issue / add FIXME comment to remove it)?

Reply:
File #218